README.rst

```diff
@@ -19,6 +19,8 @@ Hopefully this is enough to get you going::
     -v, --verbose
     -d, --dump
     -u UUID, --uuid=UUID  Camera UUID
+    -i id, --id=id        Camera Connection id
+    -c, --connect_with_id Connect using id instead of UUID (NVR Beta API)
```

Comment:
Sure, but that's just the major version. Although they definitely shouldn't be breaking something like this without a major bump, it would be quite a deal to change the version in the url.

Looks like this path will get you lots of data:
```
http://192.168.100.130:7080/api/2.0/bootstrap
```

Which includes data[0]['systemInfo'] which has a version in it. Let's just check to see if that is >= 3.2 and use the ID in that case, okay? We should probably be polling that on first connect anyway and exposing it to the client user anyway. Let me know if you want me to do that bit first and I'll be glad to so you can build on that.